core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java

```diff
@@ -205,6 +208,14 @@ public VirtualHostBuilder defaultHostname(String defaultHostname) {
         return this;
     }
 
+    /**
+     * Sets the default hostname of this {@link VirtualHost}.
+     */
+    public VirtualHostBuilder defaultContextPath(String contextPath) {
```

Comment:
What do you think of handling each `VirtualHost` separately for now? (so we can remove this method)

i.e.
Even if `ServerBuilder#contextPath == "/api"` and `VirtualHostBuilder#contextPath == "/v1"`, the virtual host services are served under `/v1` and not under `/api/v1`?

Reply:
No problem. then the PR could be simpler. 🙂